modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/portlet/action/UpdateRecordMVCResourceCommand.java

```diff
@@ -124,9 +125,22 @@ private Map<String, List<DDMFormFieldValue>> _toDDMFormFieldValues(
 				while (iterator.hasNext()) {
 					String languageId = iterator.next();
 
+					String valueJSONObjectString = valueJSONObject.getString(
+						languageId);
+
+					if (jsonObject.getString(
+							"name"
+						).equals(
+							"eventDate"
+						)) {
+
+						valueJSONObjectString = StringUtil.removeChars(
+							valueJSONObjectString, '[', ']', '"');
+					}
+
```

Comment:
@victorcdp Unfortunately this solution can cause other problems. Imagine if the user adds a New Data Definition with a text field named  `eventDate`..

![image](https://user-images.githubusercontent.com/34186632/204304188-b21e8721-98e0-4af9-a327-f1ae8eea755f.png)

With this solution we will also remove these characters from the text field, when we should just remove them from the date field.

Reply:
Can you check if we can fix the bug on the frontend side? You can start investigating this file [here](https://github.com/liferay/liferay-portal/blob/db7aab4b87b4f63f3bcdc90d409c220791e1dae8/modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/js/main.js#L642-L668).

Thanks!